devel/meson/Portfile

```diff
@@ -125,10 +123,6 @@ post-destroot {
         xinstall -d ${destroot}${vim_dir}/${d}
         xinstall -m 644 ${worksrcpath}/data/syntax-highlighting/vim/${d}/meson.vim ${destroot}${vim_dir}/${d}
     }
-
-    set emacs_lispdir ${prefix}/share/emacs/site-lisp
-    xinstall -d ${destroot}${emacs_lispdir}
-    xinstall -m 644 ${worksrcpath}/data/syntax-highlighting/emacs/meson.el ${destroot}${emacs_lispdir}
```

Comment:
why are you deleting this? presumably somebody wanted it…

Reply:
See https://github.com/mesonbuild/meson/commit/db680a78d11c45064281dbc5a158919323dda666

Reply:
oh, upstream deleted it… thanks